### PR TITLE
added more npcs to duel arena

### PR DESCRIPTION
**Pull request by jarryd229**
added more npcs to duel arena

### CODE REVIEW

File: game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/Afrah.kts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            5-> {
+                npc("cheerful", "Why did the skeleton burp?")
+                player("uncertain", "I don't know?")
+                npc("laugh", "'Cause it didn't have the guts to fart!")
+            }
+            6-> {
+                npc("talking", """
+                Did you know they think this place dates back to the
+                second age?!
+                """)
```
Comment (GregHib) [inline game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/Afrah.kts]:
```suggestion
                npc("talking", """
                    Did you know they think this place dates back to the
                    second age?!
                """)
```

File: game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/CaptainDaerkin.kts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        npc("cheerful", "Hello old chap.")
+        player("talking", """
+            What are you doing here? Shouldn't you be looking
+            after your glider?
+        """)
+        npc("cheerful", """
+            I'm pretty much retired these days old fellow. My test
+            piloting days are over. I'm just relaxing here and
+            enjoying the primal clash between man and man.
+            """)
```
Comment (GregHib) [inline game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/CaptainDaerkin.kts:21]:
```suggestion
        """)
```
I know I'm picky 🙄

File: game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/CaptainDaerkin.kts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        player("talking", """
+            You're watching the duels then. Are you going to
+            challenge someone yourself?
+        """)
+        npc("cheerful", """
+            I do find the duels entertaining to watch, but I suspect
+            that actually being involved would be a lot less fun for
+            me. I'm a lover, not a fighter!
+            """)
```
Comment (GregHib) [inline game/src/main/kotlin/world/gregs/voidps/world/map/al_kharid/duel_arena/CaptainDaerkin.kts:30]:
```suggestion
        """)
```

**Review by GregHib — CHANGES_REQUESTED** [review]
Okay yeah as there's a lot of duplication, how about a `Spectators.kts` with just talk-to's all using the same method? the other options (banking, healing etc..) can stay in their respective character scripts

File: game/src/main/kotlin/world/gregs/voidps/world/map/random_events/quiz_master.kts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import world.gregs.voidps.engine.client.ui.dialogue.talkWith
+import world.gregs.voidps.engine.entity.character.npc.NPCOption
+import world.gregs.voidps.engine.entity.character.player.Player
+import world.gregs.voidps.engine.event.on
+import world.gregs.voidps.world.interact.dialogue.type.npc
+
+on<NPCOption>({ npc.id == "quiz_master" && option == "Talk-to" }) { player: Player ->
+    player.talkWith(npc) {
+        npc("sad", "Sorry, it's usernamehere that's having the lucky day.")
```
Comment (GregHib) [inline game/src/main/kotlin/world/gregs/voidps/world/map/random_events/quiz_master.kts]: Do you want me to wait before merging so you can finish this?

Comment (jarryd229) [reply game/src/main/kotlin/world/gregs/voidps/world/map/random_events/quiz_master.kts]: iv removed it and i will re add later when i can get the info

**Review by GregHib — APPROVED** [review]

**Pull request merged**